packages/mobile/src/web3/saga.ts

```diff
@@ -148,7 +149,7 @@ export function* getOrCreateAccount() {
     Logger.debug(TAG + '@getOrCreateAccount', 'Creating a new account')
 
     const wordlist = getWordlist(yield select(currentLanguageSelector))
-    let mnemonic: string = yield call(generateMnemonic, MNEMONIC_BIT_LENGTH, null, wordlist)
+    let mnemonic: string = yield call(generateMnemonic, MNEMONIC_BIT_LENGTH, wordlist)
```

Comment:
@gastonponti I think there's been a disconnect on our plan for this part. I thought we wanted to continue to use `react-native-bip39` for *creating* the mnemonics but that we would use code from utils to *derive* the keys from those mnemonics. The benefit to using RN-Bip39 is that it runs in native code so it's more performant on low-end phones.

@cmcewen I believe you have a bit more context since IIRC you were the one to integrate RN-Bip39 into the app. Were we seeing long waits on low-end phones when generating mnemonics in JS-land?

Reply:
Yup so the issue is pbkdf2 is real slow on android in hermes. We can fix this by switching the bip39 version (https://github.com/bitcoinjs/bip39/blob/master/src/index.js#L4) similar to what we did with the rn version: https://github.com/celo-org/react-native-bip39/commit/8d007387fc8bcc18a2d0e2344ec986a0c6f15834

Reply:
Ok, I got dizzy by the bip32 talk (the one deriving), sorry.
I'm changing it right now

Reply:
The `generateMnemonic` it's fine, it's not using `pbkdf2`, but it's used to generate the seed that is input for the `bip32`.
Let me check if I could extract that, otherwise I could switch between libs (but I'm idea was to unify and reduce libs, haha)

Reply:
@gastonponti Yes, I've considered replacing our web of crypto dependencies with a single Celo one to make things more secure and auditable. Probably won't happen before mainnet though!
We can chat and figure out the best solution for now :)